### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by mesnico** [Wangt-CN/Code_CASC#3]
Thank you for the nice work. I am trying to evaluate your model on a different metric. 
Can you please share the pre-trained models? Thank you!
